app/views/users/show.html.erb

```diff
@@ -10,4 +10,26 @@
   <section>
     <h1>Bookmarked Segments</h1>
   </section>
+  <section class="github">
+    <% if current_user.token != nil %>
+    <h1>Github</h1>
+    <% @repos.first(5).each do |repo| %>
+     <div class="repo">
+    <p class="name"><%= link_to repo[:name], "http://github.com/#{repo[:full_name]}" %></p>
+    </div>
+    <% end %>
+    <h2>Followers</h2>
+      <% @followers.each do |follower| %>
+      <div class="follower">
+        <p class="follower_name"><%= link_to follower[:login], follower[:html_url] %></p>
```

Comment:
Make sure we update all of these to call on the object instead of the hash.

Reply:
We can integrate hashes after project check-in